.ci-builds/.azure-pipelines.yml

```diff
@@ -37,6 +37,19 @@ jobs:
         CC: clang
         CXX: clang++
 #-------------------------------------------------------------------------
+- job: ICU4C_Clang_Ubuntu_1604_WarningsAsErrors
+  displayName: 'C: Linux Clang WarningsAsErrors (Ubuntu 16.04)'
+  timeoutInMinutes: 30
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+    - script: |
+        export CPPFLAGS="-Wall -Werror" && cd icu4c/source && ./runConfigureICU Linux && make -j2 check
```

Comment:
Probably you only need to build, not run the tests

Reply:
Good point! I'll update to remove the check.